pylettize/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""Pylettize package."""
+
+from .pylettize import main
+
+__all__ = ["main"]
```

Comment:
This is temporary. Will probably re-export more specific things when working on #1